pybliometrics/sciencedirect/tests/test_ObjectRetrieval.py

```diff
@@ -0,0 +1,43 @@
+"""Test class ObjectRetrieval"""
+import xml.etree.ElementTree as ET
+
+from io import BytesIO
+from PIL import Image
+
+from pybliometrics.sciencedirect import init, ObjectRetrieval
+
+init()
+
+or_1 = ObjectRetrieval('S156984322300331X',
+                       'gr10.jpg',
+                       refresh=30)
+or_2 = ObjectRetrieval('10.1016/j.rcim.2020.102086',
+                       'si92.svg',
+                       id_type='doi',
+                       refresh=30)
+
+
+def test_object():
+    """Tests whether the object is a BytesIO object and its content."""
+    obj_1_last_50 = b'\xbf\xbd\xb6\xeb;+\\\x87Y7\x94[y\x17\xe3\xeb/(\xcf#\xda\xc9\x90\x80 \x08\x02\x00\x80 \x08\x02\x00\x80 \x08\x02\x00\x80 \x08\x02\x00\x80 \x08\x02\x03\xff\xd9'
+    assert isinstance(or_1.object, BytesIO)
+    assert or_1.object.getvalue()[-50:] == obj_1_last_50
+    
+    obj_2_150_200 = b"085 235.866 8.8237' width='283.039pt' xmlns='http:"
+    assert isinstance(or_2.object, BytesIO)
+    assert or_2.object.getvalue()[150:200] == obj_2_150_200
+
+
+def test_is_jpg():
+    """Tests whether the object is a JPEG image."""
+    obj_1 = or_1.object
+    with Image.open(obj_1) as img:
+        assert img.format.lower() == 'jpeg'
```

Comment:
Combine with the previous tests, because we have one test per attribute, which may test multiple things.